manageiq-providers-ibm_power_hmc.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "ibm_power_hmc", "~> 0.8.8"
+  spec.add_dependency "ibm_power_hmc", "~> 0.9.0"
```

Comment:
Is the gem update is required for this report I'll close #52 and we can do it here. If it's not related let's keep #52 as a separate change.